src/components/inputs/checkbox-input/checkbox-input.js

```diff
@@ -1,12 +1,80 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import { css } from '@emotion/core';
+import vars from '../../../../materials/custom-properties';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
 import getFieldId from '../../../utils/get-field-id';
 import Text from '../../typography/text';
 import Spacings from '../../spacings';
 import Icons from './icons';
-import styles from './checkbox-input.mod.css';
+
+const getLabelStyles = (isDisabled, hasError) => {
+  if (isDisabled) {
+    return css`
+      cursor: not-allowed;
+    `;
+  }
+  if (hasError) {
+    return css``;
+  }
+  return css`
+    cursor: pointer;
+    &:hover svg [id$='borderAndContent'] > [id$='border'] {
+      stroke: ${vars.borderColorInputFocus};
+    }
+  `;
+};
+
+const getCheckboxWrapperStyles = (isDisabled, hasError, isHovered) => {
+  const baseStyles = css`
```

Comment:
Can we hoist this out of the function and name it checkboxWrapperBaseStyles? Would avoid the variable to kind of regenerate the css on each invocation. Not sure it makes a difference in the grand scheme of things but my inner urge still sees this as a bit wasteful.